### PR TITLE
[Fix] sub partition in query

### PR DESCRIPTION
**Pull request by IHEII**
## Summary

- Fix subpartition in query

## Solution Description

- Fix subpartition in query

### CODE REVIEW

File: src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientQueryImpl.java [hunk 1]
```diff
@@ -162,6 +163,7 @@ public ObTableClientQueryStreamResult executeInternal() throws Exception {
                     end[i] = endKey.getObj(i).getValue();
                 }
                 ObBorderFlag borderFlag = rang.getBorderFlag();
+                // result of List<Pair<logicId, obTableParams>>
```
Comment (WeiXinChan) [inline src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientQueryImpl.java:166]: Maybe you should state who it is

Comment (WeiXinChan) [reply src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientQueryImpl.java:166]: pairs ？

File: src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientQueryImpl.java [hunk 2]
```diff
@@ -125,6 +125,7 @@ public ObTableClientQueryStreamResult executeInternal() throws Exception {
             throw new ObTableException("offset can not be use without limit");
         }
         final long startTime = System.currentTimeMillis();
+        // Map of <logicId, Pair<logicId, param>>
```
Comment (WeiXinChan) [inline src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientQueryImpl.java:128]: do you comment partitionObTables ?

File: src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientQueryImpl.java (hunk 1 above)
Comment (IHEII) [reply src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientQueryImpl.java]: Yes, it means that _pairs_ is a list of pair, which left element is the logicId and the right element is the param.

File: src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientQueryImpl.java (hunk 2 above)
Comment (IHEII) [reply src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientQueryImpl.java]: For readability, have changed it into
```
// partitionObTables -> Map<logicId, Pair<logicId, param>>
```

**Comment by WeiXinChan** [comment]
LGTM

**Pull request merged**